### ISSUE TITLE
Pretrained model?

### ISSUE THREAD

**Issue by HalleyYoung** [jason9693/MusicTransformer-pytorch#6]
Hi,

Would it be possible to obtain a pre-trained model?  I keep running into either vanishing or exploding gradient problems when I run.

Thanks!

**Comment by Rane90** [comment]
@HalleyYoung  Please contact me if you managed to find a pre-trained model.

**Comment by Becomebright** [comment]
@Rane90 Hi, could you please send me (shangzhe.di@gmail.com) the pre-trained model? My model somehow only generate nonsense.

**Comment by CNelias** [comment]
Up: have any of you been able to find the pre-trained version? I would also be interested!

**Comment by wangyanshanii** [comment]
@HalleyYoung  Excuse me, please contact me if you have the pre-train model ,thanks!
